### ISSUE TITLE
Getting lots of seemingly incorrect messages in kubelet log

### ISSUE THREAD

**Issue by smarterclayton** [GoogleCloudPlatform/kubernetes#5669]
```
W0319 21:03:54.108655    4549 docker.go:733] found a container with the "k8s" prefix, but too few fields (5): "k8s_test.9c53d670_test.default.api_9a1ffb8c-cd0f-11e4-9f18-080027c5bfa9_4d73b503"
```

Getting that every few seconds, and it doesn't seem like the right error.  This is on faab509a23e21afe6e4811eadf693d939c17984f

**Comment by smarterclayton** [comment]
The container was starting and failing repeatedly (it's the "php" image from docker hub).

**Comment by dchen1107** [comment]
Ahh it is known issue #5372, and should be fixed by #5395

**Comment by dchen1107** [comment]
I am closing this one as dup of #5372. Please re-open it if we still observed the issue.
